lib/drizzle/application.ex

```diff
@@ -36,4 +32,37 @@ defmodule Drizzle.Application do
       {Drizzle.TodaysEvents, []}
     ]
   end
+
+  def wait_for_internet(iface) do
```

Comment:
Was this an issue while testing? As far as I'm aware, it should bind correctly once interfaces come back up. And `mdns_lite` that's brought in with `nerves_pack`, it should also register our local domain when connection comes up as well. Id be interested to know if you're seeing otherwise 🤔

Reply:
I did this because we just needed to have port 80 unbound for the VintageNetWizard's UI to run.
There's a config option to run the wizard at a different port but I dont know if the captive portal scheme allows for setting a port. I will try this approach and see where it gets.

Reply:
ok I rolled this back and just set the wizard to run at port 81, seems to work fine.